packages/xdl/src/Versions.js

```diff
@@ -83,6 +83,16 @@ export async function newestSdkVersionAsync() {
   return result;
 }
 
+export async function nextDeprecatedSdkVersionAsync() {
+  const sdkVersions = await sdkVersionsAsync();
+  const supportedVersions =  _.pickBy(sdkVersions, v => v.isDeprecated === undefined);
```

Comment:
```suggestion
  const supportedVersions =  _.pickBy(sdkVersions, v => !v.isDeprecated);
```
(Should work also if `isDeprecated` is `false`.)